razzle.config.js

```diff
@@ -47,6 +48,13 @@ module.exports = {
             },
             updateStrategy: 'changed',
           }),
+          new CompressionPlugin({
+            algorithm: 'gzip',
+            filename: '[path].gz[query]',
+            test: /\.js$/,
+            threshold: 10240,
```

Comment:
Contradictory to above the default threshold is 0 - https://www.npmjs.com/package/compression-webpack-plugin#threshold

Reply:
True, I find these numbers repeated across all such configs with no description of why these values are used. There are obvious reasons not to set to 0 and 1 for threshold and minRatio, but why these exact numbers are used I don't know.